packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenuToggleButton.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```

Comment:
👀 - tests are passing over here now! Looking good @nicolethoen! 

I am seeing a few lint errors in this file... not quite sure though (could just be my IDE). These changes below seemed to clear them up. I also marked `fillTemplate` function param `templateString` as `Function | string` in `util.js` to match the js prop-types. You would need to run `yarn test:pf4 -u` again to update snapshots if making these changes... 

```
import React from 'react';
import PropTypes from 'prop-types';
import { CaretDownIcon } from '@patternfly/react-icons';
import styles from '@patternfly/patternfly/components/OptionsMenu/options-menu.css';
import { css } from '@patternfly/react-styles';
import { fillTemplate } from '../../helpers/util';

const OptionsMenuToggleButton = ({
  parentId,
  onToggle,
  isOpen,
  hideCaret,
  toggleTemplate: ToggleTemplate,
  templateProps
}) => (
  <button
    className={css(styles.optionsMenuToggleButton)}
    id={`${parentId}-toggle`}
    aria-haspopup="listbox"
    aria-labelledby={`${parentId}-toggle ${parentId}-label`}
    aria-expanded={isOpen}
    onClick={onToggle}
    type="button"
  >
    {ToggleTemplate && (
      <span className={css(styles.optionsMenuToggleText)}>
        {typeof ToggleTemplate === 'string' ? (
          fillTemplate(ToggleTemplate, templateProps)
        ) : (
          <ToggleTemplate templateProps={templateProps} />
        )}
      </span>
    )}
    {!hideCaret && <CaretDownIcon aria-hidden />}
  </button>
);

OptionsMenuToggleButton.propTypes = {
  /** Id of the parent Options menu component */
  parentId: PropTypes.string.isRequired,
  /** Callback for when this Options menu is toggled */
  onToggle: PropTypes.func,
  /** Flag to indicate if menu is open */
  isOpen: PropTypes.bool,
  /** Content to be rendered in the Options menu toggle button */
  toggleTemplate: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
  /** Props to be passed to the Options menu toggle button template */
  templateProps: PropTypes.object,
  /** hide the toggle caret */
  hideCaret: PropTypes.bool
};

OptionsMenuToggleButton.defaultProps = {
  onToggle: Function.prototype,
  isOpen: false,
  templateProps: undefined,
  hideCaret: true
};

export default OptionsMenuToggleButton;
```